tests/integration/aws/general/deployment/tests.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const path = require('path');
+const expect = require('chai').expect;
+const execSync = require('child_process').execSync;
+const AWS = require('aws-sdk');
+const EOL = require('os').EOL;
+const fs = require('fs');
+
+const CF = new AWS.CloudFormation({ region: 'us-east-1' });
+const Utils = require('../../../../utils/index');
+
+describe('AWS - General: Deployment with --noDeploy', function () {
+  this.timeout(0);
+  let serviceName;
+
+  before(() => {
+    serviceName = Utils.createTestService('aws-nodejs', path.join(__dirname, 'service'));
+  });
+
+  it('should deploy package with --noDeploy flag', () => {
+    const deploy = execSync(`${Utils.serverlessExec} deploy --noDeploy`);
+    const result = new Buffer(deploy, 'base64').toString();
+    const resultLines = result.split(EOL);
+    expect(resultLines[0]).to.equal('Serverless: Packaging service...');
+    expect(resultLines[1]).to.equal('Serverless: Did not deploy due to --noDeploy');
```

Comment:
This might be a little bit too specific. A check if it matches `--noDeploy` somewhere in the message might be sufficient.

Otherwise our integration test suite will break every time we change the message.